Makefile

```
@@ -96,18 +96,104 @@ release: validate test-install
 	echo "Release v$$VERSION complete. VERSION bumped to $$NEXT_VERSION."
 
+# Script inventory: which canonical scripts (in spex/scripts/) belong to which extensions
+SCRIPTS_spex := spex-flow-state.sh spex-ship-state.sh spex-ship-statusline.sh spex-finish-context.sh spex-worktree-cwd.sh spex-detach.sh
+SCRIPTS_spex_gates := spex-flow-state.sh spex-closeout-gate.sh
+SCRIPTS_spex_collab := spex-flow-state.sh spex-triage-state.sh sanitize-gh-json.py
+SCRIPTS_spex_deep_review := spex-flow-state.sh
+SCRIPTS_spex_detach := spex-detach.sh
+
+EXTENSIONS := spex spex-gates spex-collab spex-deep-review spex-detach
+
+sync-scripts:
```

checkmake (0.3.2)

[warning] Target body for "sync-scripts" exceeds allowed length of 5 lines (16).

(maxbodylength)